web-client/integration-tests/journey/petitionsClerkSubmitsPaperCaseToIrs.js

```diff
@@ -72,12 +72,18 @@ export const petitionsClerkSubmitsPaperCaseToIrs = cerebralTest => {
       CASE_STATUS_TYPES.generalDocket,
     );
     //check that documents were served
-    const documents = cerebralTest.getState('caseDetail.docketEntries');
-    for (const document of documents) {
-      if (!document.isMinuteEntry) {
-        expect(document.servedAt).toBeDefined();
-        expect(document.servedParties.length).toEqual(1);
-        expect(document.servedParties[0].role).toEqual(ROLES.irsSuperuser);
+    const docketEntries = cerebralTest.getState('caseDetail.docketEntries');
+    for (const docketEntry of docketEntries) {
+      if (!docketEntry.isMinuteEntry && !docketEntry.eventCode === 'NOTR') {
```

Comment:
Just now realized that this should be `docketEntry.eventCode !== 'NOTR'`. Fixing via merge conflict resolution here: https://github.com/ustaxcourt/ef-cms/pull/2087